tiles/src/main/java/com/protomaps/basemap/layers/Landuse.java

```diff
@@ -18,61 +19,105 @@ public static void processFeature(SourceFeature sf, FeatureCollector features, S
     if (sf.canBePolygon() && (sf.hasTag("aeroway", "aerodrome", "runway") ||
       sf.hasTag("area:aeroway", "taxiway", "runway") ||
       sf.hasTag("amenity", "hospital", "school", "kindergarten", "university", "college") ||
+      sf.hasTag("boundary", "national_park", "protected_area") ||
       sf.hasTag("landuse", "recreation_ground", "industrial", "brownfield", "railway", "cemetery", "commercial",
         "grass", "orchard", "farmland", "farmyard", "residential") ||
-      sf.hasTag("leisure", "park", "garden", "golf_course", "dog_park", "playground", "pitch") ||
+      sf.hasTag("leisure", "park", "garden", "golf_course", "dog_park", "playground", "pitch", "nature_reserve") ||
       sf.hasTag("man_made", "pier") ||
+      // TODO: (nvkelso 20230622) This use of the place tag here is dubious
       sf.hasTag("place", "neighbourhood") ||
       sf.hasTag("railway", "platform") ||
       (sf.hasTag("area", "yes") &&
-        (sf.hasTag("highway", "pedestrian", "footway") || sf.hasTag("man_made", "bridge"))))) {
+        (sf.hasTag("highway", "pedestrian", "footway") || sf.hasTag("man_made", "bridge")))))
+    {
       var poly = features.polygon(layerName)
         .setId(FeatureId.create(sf))
-        .setAttr("landuse", sf.getString("landuse"))
-        .setAttr("leisure", sf.getString("leisure"))
         .setAttr("aeroway", sf.getString("aeroway"))
-        .setAttr("area:aeroway", sf.getString("area:aeroway"))
         .setAttr("amenity", sf.getString("amenity"))
+        .setAttr("area:aeroway", sf.getString("area:aeroway"))
+        .setAttr("boundary", sf.getString("boundary"))
         .setAttr("highway", sf.getString("highway"))
+        .setAttr("landuse", sf.getString("landuse"))
+        .setAttr("leisure", sf.getString("leisure"))
         .setAttr("man_made", sf.getString("man_made"))
         .setAttr("place", sf.getString("place"))
         .setAttr("railway", sf.getString("railway"))
         .setAttr("sport", sf.getString("sport"))
-        .setZoomRange(5, 15)
+        // NOTE: (nvkelso 20230622) Consider zoom 5 instead...
+        //       But to match Protomaps v2 we do earlier
+        .setZoomRange(2, 15)
         .setMinPixelSize(3.0);
 
-      OsmNames.setOsmNames(poly, sf, 0);
-
-      if (ghostFeatures) {
-        poly.setAttr("isGhostFeature", true);
-      }
-
-      poly.setAttr("pmap:area", "");
+      // NOTE: (nvkelso 20230622) landuse labels for polygons are found in the pois layer
+      //OsmNames.setOsmNames(poly, sf, 0);
```

Comment:
No longer export names on the landuse polygons, instead we peal off named features for the POIs layer.